user-service/src/main/java/com/recipe/app/user/User.java

```diff
@@ -0,0 +1,25 @@
+package com.recipe.app.user;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document(collection = "users")
+@RequiredArgsConstructor
```

Comment:
Updated to the builder pattern. e.g.

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder

Reply:
done